internal/blockchain/ethereum/config.go

```diff
@@ -74,6 +74,8 @@ const (
 func (e *Ethereum) InitConfig(config config.Section) {
 	e.ethconnectConf = config.SubSection(EthconnectConfigKey)
 	wsclient.InitConfig(e.ethconnectConf)
+	ffresty.InitConfig(e.ethconnectConf)
```

Comment:
Why is this now needed @EnriqueL8?

Reply:
So we hide that underneath the init config from the wsclient since `wsClient.InitConfig` calls `ffresty.InitConfig`. So if we ever want to separate the configs for the ws and ffresty we would need to call it here, but since they are the same today I can remove this line